server/api.py

```diff
@@ -729,16 +760,37 @@ async def post_vcon_ingress(
 ) -> None:
     """Add vCon UUIDs to a processing chain's ingress list.
 
+    Before adding vCon UUIDs to the ingress list, ensures each vCon exists in Redis
+    by syncing from storage backends if necessary. Non-existent vCons are logged
+    as warnings and skipped, maintaining the original behavior of not throwing
+    exceptions for missing vCons.
+
     Args:
         vcon_uuids: List of vCon UUIDs to add
         ingress_list: Name of the Redis list to add the UUIDs to
 
     Raises:
-        HTTPException: If there is an error adding to the ingress list
+        HTTPException: Only for Redis operation failures, not for missing vCons
     """
     try:
         for vcon_id in vcon_uuids:
+            # Parse UUID and handle invalid format gracefully
+            try:
+                vcon_uuid = UUID(vcon_id)
+            except ValueError:
+                logger.warning(f"Invalid UUID format for vCon ID '{vcon_id}', skipping ingress list addition")
+                continue
+            
+            # Ensure the vCon exists in Redis before adding to ingress list
+            vcon = await ensure_vcon_in_redis(vcon_uuid)
```

Comment:
@pavanputhra I think it will be helpful to use mget as in some over places in that module.